pkg/apic/specparser_test.go

```diff
@@ -73,6 +79,7 @@ func TestSpecDiscovery(t *testing.T) {
 
 	// Unstructured specification
 	specParser, err = createSpecParser("./testdata/multiplication.thrift", "")
+	t.Logf("12345daleunstruct %s", err)
```

Comment:
again, all these logs needed?

Reply:
No. Ignore this PR for now. I'm trying to figure out why a few sonar-preview tests don't work on the builders.